### PR TITLE
chore:update leader-elect chart config

### PR DESCRIPTION
**Pull request by qinguoyi**
#### What this PR does / why we need it
update leader-elect chart config

#### Which issue(s) this PR fixes
none

### CODE REVIEW

File: chart/values.yaml [hunk 1]
```diff
@@ -24,7 +24,6 @@ controllerManager:
     args:
     - --health-probe-bind-address=:8081
     - --metrics-bind-address=127.0.0.1:8080
-    - --leader-elect
```
Comment (kerthcet) [inline chart/values.yaml:27]: Revert this, it's the default configuration.

File: chart/templates/deployment.yaml [hunk 2]
```diff
@@ -23,7 +23,11 @@ spec:
         kubectl.kubernetes.io/default-container: manager
     spec:
       containers:
-      - args: {{- toYaml .Values.controllerManager.manager.args | nindent 8 }}
+      - args:
```
Comment (kerthcet) [inline chart/templates/deployment.yaml:26]: Also no need to change this I think. It's already part of the controllerManager args.

File: chart/values.global.yaml [hunk 3]
```diff
@@ -1,4 +1,7 @@
 fullnameOverride: "llmaz"
 
 backendRuntime:
+  enabled: true
+
+leaderElectEnabled:
   enabled: true
```
Comment (kerthcet) [inline chart/values.global.yaml:7]: Seems we can simply add an configuration here, then it works, see example below. The snippet helps people understand how to configure the controller manager.

```
controllerManager:
  manager:
    args:
    - --health-probe-bind-address=:8081
    - --metrics-bind-address=127.0.0.1:8080
    - --leader-elect
  replicas: 1
```

Once we configured here, then run `make helm-install`, the configuration works.

**Review by kerthcet — COMMENTED** [review]
Sorry, I should explain more clear about the advices, what we need is just an example about how to configure the controller args. 

Please squash the commit after addressing the comments. Thanks.

**Comment by qinguoyi** [comment]
tk, as suggested, I re-push the code.

but, i still have a question, what is the difference between value.yaml and value.global.yaml, because there are already complete examples to config in value.yaml

**Comment by kerthcet** [comment]
> tk, as suggested, I re-push the code.
> 
> but, i still have a question, what is the difference between value.yaml and value.global.yaml, because there are already complete examples to config in value.yaml

That's a good question, the answer is values.yaml is autogenerated, if you take a look at the `Makefile`, we have a `helm` command, once run `make helm`, the values.yaml will be generated and overwritten, so we have another global values yaml to store customized configurations.

Anyway, we need to update the chart README.md for avoid such confusion.

**Comment by kerthcet** [comment]
/lgtm
/approve

Thanks @qinguoyi for your patience and efforts.

**Pull request merged**